### ISSUE TITLE
Fixes size parameter in Youtube Shorts creation template

### ISSUE THREAD

**Issue by ani1998ket** [redotvideo/examples#18]
https://github.com/redotvideo/examples/blob/a24036fc5c571b8a29fe061e5e23b5dd8e8388e4/youtube-shorts/src/project.tsx#L238

The size should updated to this, since its a youtube shorts creation template

`size: {x: 1080, y: 1920}, `

**Comment by joeskeen** [comment]
Yeah, the home page demos this "short-form video template" and has lots of images of videos in portrait mode - but then you download the template and it's a standard 16:9 wide video...

**Comment by joeskeen** [comment]
Another thing I noticed is that the README says that it was created from the Minimal Starter Template, which is also incorrect.
